launch/launch/actions/reset_launch_configurations.py

```diff
@@ -23,7 +26,7 @@
 from ..frontend import expose_action
 from ..frontend import Parser
 from ..launch_context import LaunchContext
-from ..some_substitutions_type import SomeSubstitutionsType
+from ..some_substitutions_type import SomeSubstitutionsType, SomeSubstitutionsType_types_tuple
```

Comment:
Very minor, but since the original PR did it this way, I'd prefer to keep it for consistency:

```suggestion
from ..some_substitutions_type import SomeSubstitutionsType
from ..some_substitutions_type import SomeSubstitutionsType_types_tuple
```